hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DatasourceJBossASClient.java

```diff
@@ -29,6 +30,7 @@
  * @author John Mazzitelli
  */
 public class DatasourceJBossASClient extends JBossASClient {
+    private static final Logger log = Logger.getLogger(DatasourceJBossASClient.class);
```

Comment:
cant' you juse use the protected "log" member from JBossASClient - that's what its intended purpose was.